tox.ini

```diff
@@ -16,6 +17,7 @@ description = Unit tests
 deps =
     pytest
     python_toolbox
+    pysnooper
```

Comment:
This installs the latest released version of pysnooper off PyPI. This is most probably _not_ what you want.

Note that pysnooper (off the checked out commit) is already installed by Tox automatically. Try running `pysnooper` or `python -c "import pysnooper; print(pysnooper.__version__)"` to prove it.